app/[locale]/events.tsx

```diff
@@ -0,0 +1,199 @@
+import Image from 'next/image';
+import Link from 'next/link';
+
+import { BouncyArrowButton, Button } from '~/components/buttons';
+import Heading from '~/components/heading';
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from '~/components/inputs';
+import { Card, CardContent, CardDescription, CardTitle } from '~/components/ui';
+import { getTranslations } from '~/i18n/translations';
+import { cn, getKeys } from '~/lib/utils';
+import { type events as eventsSchema } from '~/server/db';
+import { getS3Url } from '~/server/s3';
+
+export default async function Events({
+  category: currentCategory,
+  locale,
+}: {
+  category: (typeof eventsSchema.category.enumValues)[number];
+  locale: string;
+}) {
+  const text = (await getTranslations(locale)).Events;
+
+  return (
+    <article
+      className="relative bg-cover bg-no-repeat pb-32 pt-[72px] md:pb-40"
+      id="events"
+      style={{
+        backgroundImage: `url('${getS3Url()}/assets/dottedarrows-1.png')`,
+      }}
+    >
+      <Heading
+        className="container"
+        glyphDirection="ltr"
+        heading="h2"
+        href="#events"
+        text={text.title}
+      />
+
+      <article className="container relative lg:flex lg:gap-16">
+        <ol
+          className={cn(
+            'hidden lg:block',
+            'sticky top-[88px]',
+            'h-fit min-w-72 space-y-4'
+          )}
+        >
+          {getKeys(text.categories).map((category, index) => (
+            <li key={index}>
+              <Link
+                className="flex"
+                href={{ query: { eventsCategory: category } }}
+                scroll={false}
+              >
+                <Button
+                  className={cn(
+                    'flex-auto rounded-xl text-center font-serif text-shade-light',
+                    'lg:border lg:border-primary-700 lg:p-6 lg:text-2xl lg:drop-shadow-2xl',
+                    'lg:bg-neutral-50 lg:text-primary-700',
+                    'lg:hover:bg-primary-700 lg:hover:text-shade-light',
+                    category === currentCategory
+                      ? 'bg-primary-300 lg:bg-primary-700 lg:text-shade-light'
+                      : 'lg:bg-opacity-60'
+                  )}
+                >
+                  {text.categories[category]}
+                </Button>
```

Comment:
Please cleanup the styles here. This entire thing is hidden below the `lg` breakpoint, so using `lg:` for any style is the same as not using it.